doc/migration/version-upgrade-guide.md

```diff
@@ -153,9 +153,9 @@ The following sections outline the upgrade steps for each SDK version along with
   deploy/olm-catalog/
   └── memcached-operator
       ├── 0.1.0
-      │   └── memcached-operator.v0.1.0.clusterserviceversion.yaml
+      │   └── memcached-operator.v0.1.0.clusterserviceversion.yaml
       ├── 0.2.0
-      │   └── memcached-operator.v0.2.0.clusterserviceversion.yaml
+      │   └── memcached-operator.v0.2.0.clusterserviceversion.yaml
```

Comment:
Nit: This change seems inadvertant/unrelated? Looking at the file with hexdump, it looks like `master` has two non-breaking spaces (`a0 20`) and your branch has two breaking spaces (`20`). Not sure if that matters, but since this is unrelated, I'd suggested reverting these lines.